src/Controller/Organization/MissionType/MissionTypeDeleteController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Organization\MissionType;
+
+use App\Entity\MissionType;
+use App\Entity\Organization;
+use Doctrine\ORM\EntityManagerInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * @Route("/delete/{id}", name="app_organization_mission_type_delete", methods={"GET"})
+ * @Security("missionType.organization == user")
+ * @IsGranted("ROLE_PARENT_ORGANIZATION")
+ */
+class MissionTypeDeleteController extends AbstractController
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private TranslatorInterface $translator;
+
+    public function __construct(TranslatorInterface $translator)
```

Comment:
you should inject `EntityManagerInterface $entityManager` here also

Reply:
Indeed, well seen :)